content/cpp/concepts/sets/sets.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'Sets'
+Description: 'Sets are associative containers that store unique elements which can be referenced by the value of the element.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Data Types'
+  - 'Sets'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Sets are associative containers which store unique elements that can be referenced by an element's value. The value, which is itself the key to access an element in the set, is constant. Once assigned it cannot be changed. However, existing values can be removed or new values can be added.
+
+## Syntax
+
+```pseudo
+std::set<datatype> setName;
+```
+
+A set can be created by using the `set` keyword and declaring a [data type](https://www.codecademy.com/resources/docs/cpp/data-types) and name.
+
+## Setting a Different Comparison Function
+
+By default, values of the set are sorted in ascending order. The comparison function can be changed from the default to `std::greater<int>` in order to sort the values in descending order:
+
+```pseudo
+std::set<int, std::greater<int> > setName;
+```
+
+The following example shows this in use:
+
+```cpp
```

Comment:
Maybe set an example for the default behavior, then show the change in output with the comparison function example?

Reply:
@SSwiniarski I'll add that.